/**:
  ros__parameters:
    mcu_ip_addr: 192.168.9.5

/sensor_board_driver/button_battery_state:
  ros__parameters:
    sub_port: 9000
    timeout_ms: 1000

/sensor_board_driver/buzzer_switch:
  ros__parameters:
    mcu_port: 9001
    this_port: 9001

/sensor_board_driver/depth:
  ros__parameters:
    sub_port: 9002
    timeout_ms: 1000

/sensor_board_driver/environment:
  ros__parameters:
    sub_port: 9003
    timeout_ms: 1000

/sensor_board_driver/imu_reset:
  ros__parameters:
    mcu_port: 9004
    this_port: 9004

/sensor_board_driver/leak:
  ros__parameters:
    sub_port: 9005
    timeout_ms: 1000

/sensor_board_driver/rgb:
  ros__parameters:
    mcu_port: 9006
    this_port: 9006
 
/sensor_board_driver/rtc_gnss:
  ros__parameters:
    sub_port: 9007
    timeout_ms: 1000

/sensor_board_driver/rtc_time:
  ros__parameters:
    sub_port: 9008
    timeout_ms: 1000

/sensor_board_driver/tilt_servo:
  ros__parameters:
    mcu_port: 9009
    this_port: 9009

/sensor_board_driver/zed_power:
  ros__parameters:
    mcu_port: 9010
    this_port: 9010
